File: Requst Sender ID Codes/SenderIDRequest.py
import requests

url = "https://zenoapi.com/api/sms/sender-ids/"
headers = {
    "x-api-key": "your_api_key_here",
    "Content-Type": "application/json"
}
data = {
    "sender_id": "Sokabet",
    "description": "Betting Platform"
}

response = requests.post(url, json=data, headers=headers)
print(response.json())

File: Requst Sender ID/SenderIDRequest.js
const axios = require("axios");

axios.post("https://zenoapi.com/api/sms/sender-ids/", {
    sender_id: "Sokabet",
    description: "Betting Platform"
}, {
    headers: {
        "x-api-key": "your_api_key_here",
        "Content-Type": "application/json"
    }
}).then(res => {
    console.log(res.data);
}).catch(err => {
    console.error(err.response.data);
});
